recipes-bsp/u-boot/files/0013-arm-dts-Add-the-iot2050-advanced-m2-variant.patch

```diff
@@ -0,0 +1,54 @@
+From be313110d322e51c83b4649a1be48f37de2b6882 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Wed, 12 Jan 2022 10:38:28 +0800
+Subject: [PATCH 13/16] arm:dts:Add the iot2050 advanced m2 variant
+
+there is a variant of iot2050, add the support for
+this new variant
+
+Signed-off-by: chao zeng <chao.zeng@siemens.com>
+---
+ arch/arm/dts/k3-am65-iot2050-boot-image.dtsi | 20 ++++++++++++++++++++
+ 1 file changed, 20 insertions(+)
+
+diff --git a/arch/arm/dts/k3-am65-iot2050-boot-image.dtsi b/arch/arm/dts/k3-am65-iot2050-boot-image.dtsi
+index 69479d7b18..9ca8edb47a 100644
+--- a/arch/arm/dts/k3-am65-iot2050-boot-image.dtsi
++++ b/arch/arm/dts/k3-am65-iot2050-boot-image.dtsi
+@@ -81,6 +81,16 @@
+ 					};
+ 				};
+ 
++				fdt-iot2050-advanced-m2 {
++					description = "k3-am6548-iot2050-advanced-m2.dtb";
++					type = "flat_dt";
```

Comment:
this dtb need be added into the OF_LIST?

Reply:
We don't use CONFIG_OF_LIST for the IOT2050. See recent cleanup patch upstream.

Reply:
We need the of_list, otherwise the https://github.com/BaochengSu/u-boot/blob/808c79050b0a1ee2e2b04e38f3d79e3ac2483cdc/tools/binman/README.entries#L328 would not work.

Reply:
That's true, but we don't use that sequence trick yet. That would be a separate upstream cleanup commit first of all. RIght now, CONFIG_OF_LIST is unused, thus requires no maintenance.

Reply:
BTW, before adding anything M2-related here, we must update U-Boot first of all: No new features based on old versions, upstream first (at least as baseline, submit once we settled over it ourselves).

Reply:
Sure, since the hardware is not ready yet, this PR is marked as `WIP` and is indeed at a very early stage. I think Chao's initail purpose for this PR is to draw some attentions of code review.

Reply:
BTW, [this](https://github.com/BaochengSu/u-boot/commits/su/slipt-pg2) is the v2 patch of the PGS splitting, need your review comments :)